src/components/Loans/OpenLoans/components/PrintToPDFWrapper/PrintTemplate.js

```diff
@@ -1,14 +1,29 @@
+import React from 'react';
```

Comment:
This import is redundant

Reply:
Hey @artem-blazhko 
Please check the description, It is a bug fix release